apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.html

```diff
@@ -0,0 +1,120 @@
+<img class="img-bg" src="assets/background.png" />
+
+<div class="content">
+  <span class="title">Create a Cell Distance Visualization</span>
+
+  <mat-card class="creation-step">
+    <mat-card-header>
+      <mat-card-title>Format and Upload Data</mat-card-title>
+    </mat-card-header>
+    <mat-card-content> </mat-card-content>
+    <mat-card-actions>
+      <button mat-button *ngIf="!dataUploaded" (click)="upload()">Upload CSV</button>
+      <button mat-button *ngIf="dataUploaded" (click)="removeCSV()">Upload CSV</button>
+    </mat-card-actions>
+  </mat-card>
+
+  <mat-card class="creation-step">
+    <mat-card-header>
+      <mat-card-title>Optional: Select Anchor Cell type</mat-card-title>
+    </mat-card-header>
+    <mat-card-content>
+      <mat-form-field>
+        <mat-select [(value)]="cellTypes[0].value" (selectionChange)="updateAnchorCellType($event.value)">
```

Comment:
This two-way binding will deeply mutate the first item in the `cellTypes` array. This does not seem like what you might intend. I think what you what is a regular input binding `[value]=...`